modules/gapi/test/cpu/gapi_ocv_stateful_kernel_tests.cpp

```diff
@@ -50,10 +50,7 @@ namespace
         if (!initialized)
         {
             // Since G-API has no own test data (yet), it is taken from the common space
-            const char* testDataPath = getenv("OPENCV_TEST_DATA_PATH");
-            GAPI_Assert(testDataPath != nullptr);
-
-            cvtest::addDataSearchPath(testDataPath);
+            cvtest::addDataSearchPath("OPENCV_TEST_DATA_PATH");
```

Comment:
Wrong change.

environment variable's name != path

Reply:
I know. It is my mistake.